helpers/fstab.py

```diff
@@ -45,14 +46,36 @@ def __eq__(self, o):
         def __str__(self):
             return "{} {} {} {} {} {}".format(self.device, self.mountpoint, self.filesystem, self.options, self.d, self.p)
 
+        def get(self, item):
+            if not isinstance(item,basestring):
+                raise ValueError('Specified parameter {0} must be a string')
+            item = item.lower()
+            if item in self.__dict__.keys():
```

Comment:
Replace this with getattr(self, item) as it is doing the same thing
Note: this will raise an attributeerror and not a valueerror then